client/me/purchases/manage-purchase/index.jsx

```diff
@@ -431,6 +432,32 @@ const ManagePurchase = React.createClass( {
 		}
 	},
 
+	renderRemovePurchaseInformation() {
+		const purchase = getPurchase( this.props ),
+			contactSupportUrl = 'https://support.wordpress.com/';
```

Comment:
There is really no standard way, there are urls with locale and without. Both work properly. When there is no locale provided it redirects to your lang version, so in long run this seems to be better approach to let server decide which locale load.